src/site/content/en/blog/notification-triggers/index.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Notification Triggers
+subhead:
+  Notification Triggers allows you to schedule local notifications that don't require a network
+  connection, which makes them an ideal solution for use cases like calendar apps.
+authors:
+  - thomassteiner
+description:
+  The Notification Triggers API allows developers to schedule local notifications that don't require
+  a network connection, which makes them an ideal solution for use cases like calendar apps.
+date: 2019-10-24
+updated: 2019-10-24
+tags:
+  - post
+  - capabilities
+  - fugu
+  - behind-a-flag
+  - notification-triggers
+---
+
+{% Aside %}
+  The Notification Triggers API, part of Google's
+  [capabilities project](https://developers.google.com/web/updates/capabilities),
+  is currently in development. This post will be updated as the
+  implementation progresses.
+{% endAside %}
+
+## What are Notification Triggers? {: #what }
+
+Web developers have the ability to display notifications using the
+[Web Notifications API](https://www.w3.org/TR/notifications/). This feature is often used in
+collaboration with the [Push API](https://w3c.github.io/push-api/) to inform the user of
```

Comment:
Delete 'in collaboration'.

Same reason.

Reply:
Done.